cfgov/unprocessed/css/molecules/form-field-with-button.less

```diff
@@ -31,33 +31,37 @@
     - cfgov-molecules
 */
 
-
-
 // TODO: Look into merging email-signup and form-field-with-button styles
 // Form field with button isn't used anywhere else
 .m-form-field-with-button {
     .m-form-field {
-        margin-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em );
+        margin-bottom: 0;
     }
 
-    &_info {
-        margin-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em );
+    &_info,
+    .a-btn {
+        margin-top: unit( (@grid_gutter-width / 2) / @base-font-size-px, em );
```

Comment:
Super minor, check the spaces within parens here and throughout the less files.

Reply:
Good catch! Fixed in https://github.com/cfpb/cfgov-refresh/pull/4157/commits/00c1a2215cfefb41ef522fe6b61ffe0e797b12ba